x-pack/plugins/security_solution/public/common/components/item_details_card/index.tsx

```diff
@@ -52,23 +52,30 @@ const DetailsSection = styled(EuiFlexItem)`
 `;
 
 const DescriptionListTitle = styled(EuiDescriptionListTitle)`
-  width: 40%;
+  &&& {
```

Comment:
I remember coming across this once, but forgot what it does. Is it applying the style to "self"?

Reply:
increases specificity N times. Basically generates .class.class.class